report_csv/__manifest__.py

```diff
@@ -8,9 +8,8 @@
     "category": "Reporting",
     "version": "16.0.2.0.0",
     "license": "AGPL-3",
-    "depends": ["base", "web"],
+    "depends": ["base", "web", "report_format_option"],
```

Comment:
This is incorrect, as you can't change dependencies of a merged module.

Reply:
@pedrobaeza Do you mean that this goes against the stable policy and report_csv should stay as is?  In that case, we will try to do this in the 17.0 migration, and we may just propose to add report_format_option in 16.0 anyway.

Reply:
Yes, you can't do it on an existing version, or you will break existing installations. I also see that the extracted code is not worth for such "reusing", but that's only my opinion.

Reply:
Fine by me. We need report_format_option anyway for a client to apply some format conversions on text reports. That's why we proposed to split the common feature off report_csv to avoid redundancy. We will try to do this in 17.0.

@AungKoKoLin1997 Can you please close this PR and create another one just to add report_format_option?